framework/src/modules/random/stores/used_hash_onions.ts

```diff
@@ -100,6 +100,16 @@ export class UsedHashOnionsStore extends BaseOffchainStore<UsedHashOnionStoreObj
 
 		if (index === -1) {
 			originalUsedHashOnions.push(usedHashOnion);
+			// if newly inserted entry has count 0, then cleanup previous entries
+			if (usedHashOnion.count === 0) {
+				const newUsedHashonions = [usedHashOnion];
```

Comment:
Variable `newUsedHashonions` is not properly camelCased. Should be `newUsedHashOnions`